datafusion/src/execution/mod.rs

```diff
@@ -20,6 +20,6 @@
 pub mod context;
 pub mod dataframe_impl;
 pub(crate) mod disk_manager;
-pub(crate) mod memory_manager;
+pub mod memory_manager;
```

Comment:
I also had to make `MemoryManager` public in https://github.com/apache/arrow-datafusion/pull/1668 -- I went with just the `MemoryManager` structure, but maybe making the whole module would be better 🤔